BitmarkRegistry/AppDelegate.swift

```diff
@@ -103,7 +104,7 @@ private extension AppDelegate {
   // Create a Sentry client and start crash handler
   func initSentry() {
     do {
-      Client.shared = try Client(dsn: "https://92d49f612d5f4cd89427cef0cd39794f@sentry.io/1494841")
+      Client.shared = try Client(dsn: "https://b3e51817355b4b35b37bff03ab51256b@sentry.io/1496699")
```

Comment:
What is this dsn?

Reply:
Sorry, mistake for push this line of code.
(This is my personal Sentry account - cause Bitmark Sentry account is currently exhausted its event quota; and also I don't want to make noise people for my testing process :P)